docs/source/customizations/overview.mdx

```diff
@@ -18,7 +18,7 @@ The Apollo Router supports two types of plugins:
 
 If your plugin only needs to make basic changes to request or response headers, we recommend first checking whether a [Rhai script](./rhai/) can accomplish what you need.
 
-If not, the [hello world](https://github.com/apollographql/router/blob/225915f6e084b13c3a0972f170cf4a1a6f8e7f28/examples/hello-world/src/hello_world.rs) example plugin is a helpful starting point for writing your first [native Rust plugin](./native/).
+If not, the [hello world](https://github.com/apollographql/router/blob/2b094d2eb47feab3f524bae53215a9c1c2cd0bea/examples/hello-world/src/hello_world.rs) example plugin is a helpful starting point for writing your first [native Rust plugin](./native/).
```

Comment:
I think it could be better to have a relative links instead. It will be easier to maintain

Reply:
@StephenBarlow Is it policy to use links like this? It meant that when I navigated to the examples directory after everything was out of date because it was stuck on an old hash.

Reply:
@BrynCooke I don't know that it's policy, but if files move around, then branch-based links and up broken too.  I'm not sure there's a happy path in all cases, but for **this particular URL**, I think it would be okay to expect that to be a known location and I'd support using `HEAD` or `main` in place of the ref.  (The benefit of `HEAD` is that will also continue to work if the default branch name is renamed.)

Reply:
Yes, I don't see this particular URL changing ever.